microsoft-edge/webview2/how-to/print.md

```diff
@@ -203,14 +203,14 @@ async void PrintToDefaultPrinter(object sender, RoutedEventArgs e)
 
   try
   {
-    // Passing null for `PrintSettings` results in default print settings used.
-    // Prints current web page with the default page and printer settings.
+    // Prints the current webpage by using the default printer settings and page settings.
+    // Passing null for PrintSettings causes the default print settings to be used.
     CoreWebView2PrintStatus printStatus = await WebView2.CoreWebView2.PrintAsync(null);
 
     if (printStatus == CoreWebView2PrintStatus.Succeeded)
     {
       dialog = new MessageDialog("Printing " + title +
```

Comment:
@monica-ch - fyi this revision changes a line of code, but merely a displayed string.